src/jaxsim/parsers/descriptions/model.py

```diff
@@ -112,7 +115,7 @@ def build_model_from(
             final_collisions.append(new_collision_shape)
 
             # If the frame was found, update the collidable points' pose and add them
-            #  to the new collision shape
+            # to the new collision shape
```

Comment:
Just if you want to be coherent with the rest of the comments style
```suggestion
            # to the new collision shape.
```